demisto_sdk/commands/generate_docs/generate_integration_doc.py

```diff
@@ -220,6 +221,97 @@ def generate_setup_section(yaml_data: dict):
     return section
 
 
+# Incident Mirroring
+
+def is_configuration_exists(yml_data: dict, display: str):
+    """
+    Args:
+        yml_data: yml data of the integration
+        display: configuration to search
+
+    Returns:
+        the cong if display exists in the yml_data, else none.
+
+    """
+
+    for conf in yml_data.get('configuration', []):
+        if conf.get('display') == display:
+            return conf
+    return None
+
+
+def generate_mirroring_section(yaml_data: dict):
+    """
+
+    Args:
+        yaml_data: dict representing the yml file of the integration.
+
+    Returns: markdown section of Incident Mirroring.
+
+    """
+    integration_name = format(yaml_data['display'])
+    directions = {
+        'None': 'Turns off incident mirroring.',
+        'Incoming': 'Any changes in CrowdStrike Falcon incidents (state, status, tactics, techniques, objectives, '
+                    'tags, hosts.hostname) will be reflected in XSOAR incidents.',
+        'Outgoing': 'Any changes in XSOAR incidents will be reflected in CrowdStrike Falcon incidents (tags, status).',
+        'Incoming And Outgoing': 'Changes in XSOAR incidents and CrowdStrike Falcon incidents will be reflected in both directions.'
```

Comment:
```suggestion
        'Incoming': 'Any changes in CrowdStrike Falcon incidents (%%mirroring incoming fields%%) will be reflected in XSOAR incidents.',
        'Outgoing': 'Any changes in XSOAR incidents will be reflected in CrowdStrike Falcon incidents (%%mirroring outgoing fields%%).',
        'Incoming And Outgoing': 'Changes in XSOAR incidents and CrowdStrike Falcon incidents will be reflected in both directions.'
```
Here change `CrowdStrike Falcon` with integration name, and the fields names should be added by the programmer.